semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/ErrorIncubatingAttributes.java

```diff
@@ -47,13 +54,11 @@ public final class ErrorIncubatingAttributes {
    * Values for {@link #ERROR_TYPE}.
    *
    * @deprecated deprecated in favor of stable {@link
-   *     io.opentelemetry.semconv.ErrorAttributes.ErrorTypeValues} attribute.
+   *     io.opentelemetry.semconv.ErrorAttributes#ERROR_TYPE} attribute.
```

Comment:
nit: this used to point to the enum type and now points to the attribute name.

Reply:
Fixed